index.html

```diff
@@ -1901,100 +1901,216 @@
           <dfn>country</dfn> attribute
         </h2>
         <p>
```

Comment:
Above this we have "When called, run" for toJSON. To stay consistent in the tenses used, we should change that to "When called, runs".

Reply:
fixed both .toJSON() instances